modules/transactions/src/main/java/org/apache/ignite/internal/tx/LockManager.java

```diff
@@ -36,14 +36,15 @@ public interface LockManager extends EventProducer<LockEvent, LockEventParameter
      * @param lockMode Lock mode, for example shared, exclusive, intention-shared etc.
      * @return The future with gained lock that will be completed when a lock is successfully acquired.
      */
-    public CompletableFuture<Lock> acquire(UUID txId, LockKey lockKey, LockMode lockMode);
+    CompletableFuture<Lock> acquire(UUID txId, LockKey lockKey, LockMode lockMode);
 
     /**
      * Attempts to release the specified lock.
      *
      * @param lock Lock to release.
      */
-    public void release(Lock lock);
+    @TestOnly
```

Comment:
It's also called from within org.apache.ignite.internal.tx.impl.HeapUnboundedLockManager#releaseAll.
BTW it's strange. Does it mean that we don't have checks that verify that @TestOnly code is only used from within tests?

Reply:
I don't know. It is called internally from production method, so looks OK.